opal/mca/common/cuda/common_cuda.c

```diff
@@ -300,6 +303,15 @@ void mca_common_cuda_register_mca_variables(void)
                                  MCA_BASE_VAR_SCOPE_READONLY,
                                  &mca_common_cuda_cumemcpy_timing);
 #endif /* OPAL_ENABLE_DEBUG */
+
+#if OPAL_CUDA_GPU_MEM_WAR
+    (void) mca_base_var_register("ompi", "mpi", "common_cuda", "gpu_mem_check_war",
+                                 "Set to 1 to enable GPU memory check war",
```

Comment:
WAR is used internally here.  It is short for workaround.  Updated commit will have better message.